### ISSUE TITLE
consider adding draws or to throw an error when used in posterior_predict

### ISSUE THREAD

**Issue by bnicenboim** [paul-buerkner/brms#797]
Both rstanarm and brms have `posterior_predict` with different arguments, and sometimes do the same.
It would be useful that when I use `draws`, for example, instead of `nsamples` in `posterior_predict`, it would either accept it or throw an error. Now, it silently ignores it.

**Comment by paul-buerkner** [comment]
I agree. we plan to align the function signatures in the next major release of these packages.
